### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by VadzimKavaleuski**
ask for merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+Role Variables
+--------------
+
+You can override any variable below by setting "variable: value" in playbook.
+
+
+- `heartbeat_version`
+Is used to select main Heartbeat branch to be installed (5.x or 6.x current stable versions). Default value is `6`.
+- `heartbeat_last_version`
+Is used to select specific Heartbeat version to be installed. Default value is `6.6.0`
```
Comment (tgadiev) [inline README.md]: Fix this

File: README.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+- Prepare server (add elastic repo)
+- [Optional] Create folder(s) for custom paths
+- Install heartbeat
+- Copy configuration file
+
+Requirements
+------------
+
+- Minimal Version of the ansible for installation: 2.7
```
Comment (tgadiev) [inline README.md]:
```suggestion
- Minimal Version of the ansible for installation: 2.8
```

File: vars/systemv.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+#hb_service_script: '{{ heartbeat_service_name }}'
```
Comment (tgadiev) [inline vars/systemv.yml]: Why do we need this?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (VadzimKavaleuski) [reply README.md]: code changed

File: vars/systemv.yml (hunk 3 above)
Comment (VadzimKavaleuski) [reply vars/systemv.yml]: agree with this
removed

**Review by tgadiev — APPROVED** [review]

**Pull request merged**